src/wpublish/QueriesAndChunks.cc

```diff
@@ -240,6 +181,9 @@ void QueriesAndChunks::finishedTask(wbase::Task::Ptr const& task) {
             (*_newlyDeadQueries)[qId] = stats;
         }
     }
+    if (task->isBooted()) {
+        _bootedTaskTracker.removeTask(task);
```

Comment:
This is a bit confusing to me. If the task is "booted" then why should it be removed from the tracker? I would expect the opposite.

Reply:
The tracker only tracks "booted" tasks that are actively running as it's real purpose is to estimate how much of a drain on the system they are. Once the booted tasks finish, they are no longer a drain on the system.